src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -55,58 +55,59 @@ using namespace Gaffer;
 namespace
 {
 
-template<typename T>
-std::string compoundNumericPlugRepr( const T *plug )
+std::string maskedCompoundNumericPlugRepr( const ValuePlug *plug, unsigned flagsMask )
 {
-	return maskedCompoundNumericPlugRepr( plug, Plug::All );
-}
+	std::string extraArgs = "";
 
-template<typename T>
-class CompoundNumericPlugSerialiser : public ValuePlugSerialiser
-{
-  public:
-
-	static std::string repr( const Gaffer::ValuePlug *plug, unsigned flagsMask, const std::string &extraArguments, const Serialisation *serialisation )
+	if( const V3fPlug *v3fPlug = IECore::runTimeCast<const V3fPlug>( plug ) ) 
```

Comment:
This cast still isn't right, because it's preventing us from dealing with interpretations for V2fPlug etc. In fact we shouldn't need this cast at all - we know this function should only be instantiated for types T where T is a CompoundNumericPlug of some sort, so we should just be able to call `plug->interpretation()` and get on with it.